### DISCUSSION TITLE
autocmd的nvim-tree自动命令好像不起作用

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#105] [Q&A] [answered]
autocmd的nvim-tree自动命令是当离开了nvim-tree窗口，然后nvim-tree窗口自动关闭吗，如果是的话，不知道我这里为啥不起作用

**Comment by nshen** [comment, marked as answer]
不是离开nvim-tree窗口，是所有文件都关了就退出，有些小问题不是很好用，我自己暂时注释掉了，来自这里：
https://github.com/kyazdani42/nvim-tree.lua/discussions/1115

**Comment by HUAHUAI23** [comment]
哦哦 原来这样，我以后焦点离开窗口 哈哈哈
